src/components/mdx-inline-link/index.tsx

```diff
@@ -0,0 +1,14 @@
+import classNames from 'classnames'
+import { AnchorHTMLAttributes } from 'react'
+import s from './mdx-inline-link.module.css'
+
+// styled anchor element for tutorial and docs content
+export default function MdxInlineLink(
```

Comment:
🙌 This is great, feels like a nice step towards more unified custom MDX components